engine/access/access_test.go

```diff
@@ -461,6 +462,11 @@ func (suite *Suite) TestGetExecutionResultByBlockID() {
 
 				assert.Equal(suite.T(), marshalledEvent, er.ServiceEvents[i].Payload)
 			}
+			parsedExecResult, err := convert.ExecResultProtoToFlowExecResult(resp.ExecutionResult)
+			require.NoError(suite.T(), err)
+			assert.Equal(suite.T(), parsedExecResult, executionResult)
+			assert.EqualValues(suite.T(), parsedExecResult, executionResult)
```

Comment:
isn't `EqualValues` redundant in this case since `Equal` asserts follow more strict rules